### ISSUE TITLE
Git Command Error Help

### ISSUE THREAD

**Issue by lightandlense** [ThereforeGames/unprompted#45]
When I try and install Unprompted this is what I'm getting as the error. Suggestions?

GitCommandError: Cmd('git') failed due to: exit code(128) cmdline: git clone -v https://github.com/ThereforeGames/unprompted D:\Dreambooth 2\stable-diffusion-webui\tmp\unprompted stderr: 'Cloning into 'D:\Dreambooth 2\stable-diffusion-webui\tmp\unprompted'... POST git-upload-pack (185 bytes) POST git-upload-pack (227 bytes) Downloading lib/stable_diffusion/clipseg/weights/rd16-uni.pth (1.1 MB) Error downloading object: lib/stable_diffusion/clipseg/weights/rd16-uni.pth (61545cd): Smudge error: Error downloading lib/stable_diffusion/clipseg/weights/rd16-uni.pth (61545cdb3a28f99d33d457c64a9721ade835a9dfbda604c459de6831c504167a): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access. Errors logged to 'D:\Dreambooth 2\stable-diffusion-webui\tmp\unprompted\.git\lfs\logs\20230107T231111.9983515.log'. Use `git lfs logs last` to view the log. error: external filter 'git-lfs filter-process' failed fatal: lib/stable_diffusion/clipseg/weights/rd16-uni.pth: smudge filter lfs failed warning: Clone succeeded, but checkout failed. You can inspect what was checked out with 'git status' and retry with 'git restore --source=HEAD :/' '

**Comment by kingsushi001** [comment]
I'm getting the same error

**Comment by TheHinky** [comment]
Same here, just looks like billing issue.

**Comment by theesfeld** [comment]
same

**Comment by rokkeds** [comment]
hey folks, 
try this guide: https://stackoverflow.com/questions/62905325/this-repository-is-over-its-data-quota-account-responsible-for-lfs-bandwidth-sh

worked for me

**Comment by lightandlense** [comment]
> 

So I went through the steps in the guide and downloaded the zip from my archived but not sure really where to put the files. Suggestions?

**Comment by rokkeds** [comment]
Simply put it into stable-diffusion-webui/extensions diectory

**Comment by ThereforeGames** [comment]
Hi folks,

I responded to this matter here: https://github.com/ThereforeGames/unprompted/issues/49#issuecomment-1401312969

TL;DR the issue has been resolved in v5.1.0 by removing the ClipSEG weight files.
